### PR TITLE
Add setTags method

### LINKED ISSUE #25: Bug in ClientTracer::startSpan
When passing fields to `ClientTracer::startSpan`, if those fields includes a `tags` key, [`startSpan` attempts to call `ClientSpan::setTags`](https://github.com/lightstep/lightstep-tracer-php/blob/9990c076819baa77d40f0eee58b272048cf61e3e/lib/Client/ClientTracer.php#L230-L232), but that method doesn't exist. What it should probably do instead is `foreach` over the tags and call [`ClientSpan::setTag`](https://github.com/lightstep/lightstep-tracer-php/blob/9990c076819baa77d40f0eee58b272048cf61e3e/lib/Client/ClientSpan.php#L98) with each of the tags.

### PR DESCRIPTION
**Pull request by austinlparker**
This is intended to fix #25. If an array of tags is passed, we iterate and set individual tags.

### CODE REVIEW

File: lib/Client/ClientSpan.php [hunk 1]
```diff
@@ -95,6 +95,13 @@ public function setEndUserId($id) {
         return $this;
     }
 
+    public function setTags($fields) {
+        foreach ($fields as $item) {
```
Comment (lord2800) [inline lib/Client/ClientSpan.php]: So if I'm reading this correctly, a call to `setTags` would look like:

```
$span->setTags([['tag1', 'value1'], ['tag2', 'value2'], ...]);
```

It looks to me like the intended usage of the method is:
```
$span->setTags(['tag1' => 'value1', 'tag2' => 'value2', ...]);
```

If that's the case, this loop would need to look like:

```
foreach ($fields as $key => $value) {
```

If I'm wrong, feel free to ignore this.

Comment (austinlparker) [reply lib/Client/ClientSpan.php]: I kinda guessed as to how people were passing in multi-value tags, but based on the docs, it looked like you're expected to call `startSpan` like this: `$tracer->startSpan("test", array('tags' => array( 'foo' => 'bar', 'baz' => 'quuz')));` Is that not how you'd do it?

Comment (lord2800) [reply lib/Client/ClientSpan.php]: That's exactly how we tried to use it.

Comment (austinlparker) [reply lib/Client/ClientSpan.php]: Ah, then I think the way I've got it implemented now works then, based on the tests I added?

Comment (lord2800) [reply lib/Client/ClientSpan.php]: I don't think it's doing what you think it's doing. I wrote a quick test to verify my thoughts, like so:
```
    public function testSpanFieldsHaveCorrectKeysAndValues() {
        $tracer = LightStep::newTracer("test_group", "1234567890");
        $span = $tracer->startSpan("multi_attribute_span", array('tags' => array( 'foo' => 'bar', 'baz' => 'quuz')));
        $tags = $this->peek($span, "_tags");
        $this->assertArrayHasKey('foo', $tags);
        $this->assertSame('bar', $tags['foo']);
        $this->assertArrayHasKey('baz', $tags);
        $this->assertSame('quuz', $tags['baz']);
        $span->finish();
    }
```

What came out was that it was using the input tag value as an array, meaning it uses the first character of the input string as the tag name and the second character of the string as the tag value. For example, in this test, `$tags` has a structure that looks like: `['b' => 'a', 'q' => 'u']`.

I think the `foreach` is supposed to look like:
```
foreach ($fields as $key => $value) {
    $this->setTag($key, $value);
}
```

Comment (austinlparker) [reply lib/Client/ClientSpan.php]: 🤦‍♂ ah, i get it now. yeah, fixing this now, thanks!

Comment (lord2800) [reply lib/Client/ClientSpan.php]: This looks like I'd expect now!

**Review by MikeGoldsmith — APPROVED** [review]

**Pull request merged**